sunpy/map/sources/hinode.py

```diff
@@ -98,6 +98,20 @@ def processing_level(self):
             return
         return int(lvl)
 
+    @property
+    def unit(self):
+        # XRT data values are in DN and are converted into DN/s if the data has been normalized.
+        # A tag starting with "XRT_RENORMALIZE" is added to the HISTORY tag in that case.
+        # Ref. Section 2.11 of the XRT Analysis Guide.
+        unit = self.meta.get('BUNIT')
+        if not unit:
+            unit = u.astrophys.dn
+            hvalues = self.meta.get('HISTORY')
+            for value in hvalues:
+                if "XRT_RENORMALIZE" in value:
+                    unit = u.astrophys.dn / u.second
```

Comment:
Once #6728 is merged, this can become,

```suggestion
            if "XRT_RENORMALIZE" in self.meta.get('HISTORY')
                    unit = u.ct / u.second
```

Reply:
hi will, is `u.ct` here the same as `DN`? :eyes:

Reply:
No but there is a technical reason why we use ct instead of DN.